packages/picasso/src/RatingThumbs/RatingThumbs.tsx

```diff
@@ -0,0 +1,132 @@
+import { makeStyles, Theme } from '@material-ui/core/styles'
+import { BaseProps, SizeType } from '@toptal/picasso-shared'
+import classNames from 'classnames'
+import React, { ChangeEvent, forwardRef, useCallback, useState } from 'react'
+
+import { Container } from '../Container'
+import { ThumbsDown16, ThumbsDown24, ThumbsUp16, ThumbsUp24 } from '../Icon'
+import styles from './styles'
+
+type Size = SizeType<'small' | 'large'>
+
+export interface Props extends BaseProps {
+  /**
+   * Name of the form input group
+   */
+  name: string
+
+  /**
+   * Current value for the rating, true means thumbs up, false thumbs down and undefined means not selected
+   */
+  value?: boolean
+
+  /**
+   * Callback invoked when the rating is changed
+   */
+  onChange?: (value: boolean, event: ChangeEvent<HTMLDivElement>) => void
+
+  /**
+   * If the component should respond to user interactions. If false the value cannot be changed by the user. Defaults to true
+   */
+  interactive?: boolean
+
+  /**
+   * Size of the input control. Defaults to small
+   */
+  size?: Size
+
+  // For testing
+  dataIds?: {
```

Comment:
```suggestion
  testIds?: {
```

to be consistent with rest of codebase